universal-application-tool-0.0.1/app/services/program/ProgramQuestionDefinition.java

```diff
@@ -18,7 +18,6 @@
   @JsonProperty("id")
   public abstract long id();
 
-  @JsonIgnore
   abstract Optional<QuestionDefinition> questionDefinition();
```

Comment:
This one might actually be necessary, I think.

Reply:
I see. TIL.

I know it takes `getXXX` and `isXXX`. and I thought `XXX` itself (with no arguments) is considered a field.

Reply:
A good way to verify is to see /dev/seed. Would you give it a quick check?

Reply:
This is what I when I `bin/run-dev` and check `dev/seed`. Nothing but the `id` is stored for each question here, which is expected.
```
  "blockDefinitions" : [ {
    "id" : 1,
    "name" : "Block 1",
    "description" : "name and favorite color",
    "repeaterId" : null,
    "hidePredicate" : null,
    "optionalPredicate" : null,
    "questionDefinitions" : [ {
      "id" : 1
    }, {
      "id" : 2
    } ]
    ```

Reply:
Nice! Thanks for checking.